packages/definitions/src/emptysuperjson.ts

```diff
@@ -0,0 +1,4 @@
+import superjson from "superjson";
+
+export const emptySuperJSON = superjson.stringify({});
```

Comment:
Replace `superjson.stringify({})` with `'{"json": {}}'`
and then regenerate the mysql & sqlite migration to no longer have the update of server-settings